pulp.spec

```diff
@@ -815,6 +815,13 @@ Requires(postun): /usr/sbin/semodule
 %description    selinux
 SELinux policy for Pulp's components
 
+%pre selinux
+# Record old version,if any, so we can limit which restorecon statement are executed later
+test -e %{_localstatedir}/lib/rpm-state/%{name} || mkdir -p %{_localstatedir}/lib/rpm-state/%{name}
+oldversion=$(semodule -l | grep pulp-server)
+echo ${oldversion:12} > %{_localstatedir}/lib/rpm-state/%{name}/old-version
+
+exit 0
```

Comment:
According to https://fedoraproject.org/wiki/Packaging:ScriptletSnippets#Syntax it does need to be there.